Tiltfile

```diff
@@ -0,0 +1,65 @@
+# -*- mode: Python -*-
+
+# Extensions 
+load('ext://helm_resource', 'helm_resource', 'helm_repo')
+
+# Specifies that Tilt is allowed to run against the specified k8s context name. 
+allow_k8s_contexts('kubernetes-admin@kubernetes')
+# Anonymous, ephemeral image registry.
+default_registry('ttl.sh')
+
+# Run a defined set of services via config.set_enabled_resources.
+config.define_string_list("to-run", args=True)
+cfg = config.parse()
+groups = {
+    'balloons': ['image-build-logs', 'controller-logs'],
+    'topology-aware': ['image-build-logs', 'controller-logs'],
+}
+
+resources = []
+for resource_name in cfg.get('to-run', []):
+    if resource_name in groups:
+        resources += groups[resource_name]
+config.set_enabled_resources(resources)
+
+# Fail if the policy name is not provided.
+if len(resources) == 0:
+    fail("🚨 ERROR: No policy passed! Please run: tilt up <POLICY_NAME>")
+
+POLICY = resource_name
+IMAGE_BASE = "ttl.sh/ghcr.io/containers/nri-plugins/nri-resource-policy-"
+IMAGE = IMAGE_BASE + POLICY
+COMPILE_CMD = (
+    'cd cmd/plugins/' + POLICY +
+    '; CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -x -o ../../../build/bin/nri-resource-policy-' + POLICY + ' ./'
```

Comment:
I'm fine with an in-repo Tiltfile, if you want to use tilt and this helps you. But if we do so, I very much want the Tiltfile to build binaries and images the 'official way', using our provided build infra instead of trying to repeat the necessary steps here manually... which looks like just asking for unnecessary maintenance trouble to me.

Reply:
So for instance, use something like `make BINARIES= OTHER_IMAGE_TARGETS= PLUGINS=nri-resource-policy-+POLICY  build-plugins` here, if you only want to build a single policy binary.